features/step_definitions/block-consolidation-page-steps.js

```diff
@@ -0,0 +1,92 @@
+import { When, Then } from 'cucumber';
+import { expect } from 'chai';
+
+const BLOCK_CONSOLIDATION_COMPONENT = '.BlockConsolidationStatus_component';
+const BLOCK_CONSOLIDATION_EXPLANATION = '.BlockConsolidationStatus_content p:nth-child(3)';
+const EPOCHS_CONSOLIDATION_STATUS = '.BlockConsolidationStatus_epochs p span';
+const EPOCHS_CONSOLIDATED = '.BlockConsolidationStatus_indicatorEpochsConsolidated p';
+const TRAILING_BY_2_EPOCH = '.BlockConsolidationStatus_indicatorEpochsBehind p';
+const MAXIMUM_EPOCH = '.BlockConsolidationStatus_fullEpoch';
+const SYNC_PROGRESS = '.BlockConsolidationStatus_indicatorEpochsSynced p span';
```

Comment:
Please wrap those constants into a single object to make it clearer that all of these constants are selectors eg:
```
const SELECTORS = {
  BLOCK_CONSOLIDATION_COMPONENT: '.BlockConsolidationStatus_component',
  …
}
```

which can then be used like `SELECTORS.BLOCK_CONSOLIDATION_COMPONENT`